lib/MadNLPGPU/src/MadNLPGPU.jl

```diff
@@ -1,12 +1,22 @@
 module MadNLPGPU
 
-import CUDA: CUBLAS, CUSOLVER, CuVector, CuMatrix, toolkit_version, R_64F, has_cuda
+# CUDA
+import CUDA: CUBLAS, CUSOLVER, CuVector, CuMatrix, CuArray, toolkit_version, R_64F, has_cuda
+# Kernels
+using KernelAbstractions
+using CUDAKernels
+using LinearAlgebra
+using MadNLP
```

Comment:
I confess that I prefer to use `using` or `import` rather than importing the individual functions. This is of course a personal opinion, and can be argued about. My arguments are : 
1. I don't think there is a difference in the import time. For instance, if we take a large package like `CUDA.jl`, in three different fresh julia sessions: 
    - Explicit using
    ```julia
     julia> @time using CUDA
      3.448522 seconds (8.31 M allocations: 614.634 MiB, 0.25% compilation time)
    ```
    - Import 
    ```julia
    julia> @time import CUDA
      3.352790 seconds (8.31 M allocations: 614.756 MiB, 6.00% gc time, 0.16% compilation time)
    ```
    - Import specific function directly: 
    ```julia
    julia> @time import CUDA: CuArray
      3.238898 seconds (8.31 M allocations: 614.760 MiB, 5.40% gc time, 0.16% compilation time)
    ```

2. Usually, I prefer to use only `import Foo`, to avoid polluting the name space. And then use directly the functions of the package with `Foo.bar1`, `Foo.bar2` ... If instead we are doing `import Foo: bar1, bar2`, I find that it's difficult to understand where a particular function comes from.

But again, I am open to the discussion (and I don't know if what I am doing is on par with the recommendation from the Julia community).

Reply:
Hey @frapac, I think the reason for importing individual functions is mostly from the second reason, not polluting the namespace. Honestly, I decided to import individual functions without much doubt because @ccoffrin suggested so. But I think this is what julia style guide recommends to do: https://jump.dev/JuMP.jl/stable/developers/style/#using-vs.-import
Though I think we can use `using foo: bar` more often than `import foo:bar`, since we only need import when we extend the function.

But I also have found that sometimes this can cause confusion. For example, when I import `NLPModels.Counters`, we already had a type with the same name in `MadNLP`. In this case, I used `import Foo: bar as baz` to avoid the conflict.

Reply:
I see! In that case, how about using directly?
```julia
import Foo
x = Foo.bar()
```
I don't know if I agree with that statement in JuMP's style guide ^^
```
Prefer using ModuleName: x, p to import ModuleName.x, ModuleName.p and import MyModule: x, p because the import versions allow method extension without qualifying with the module name.
```

Reply:
This approach looks good to me :)